examples/src/bin/simple_0rtt_client.rs

```diff
@@ -1,5 +1,7 @@
 //! This is an example client that uses rustls for TLS, and sends early 0-RTT data.
 //!
+//! Usage: cargo r --bin simple_0rtt_client --package rustls-examples [domain name] [port] [path/to/ca.cert]
```

Comment:
I have a feeling that a `--` might be needed here, after `--package rustls-examples`?